core/src/com/unciv/ui/pickerscreens/PantheonPickerScreen.kt

```diff
@@ -29,7 +31,7 @@ class PantheonPickerScreen(
         }
 
         setOKAction("Choose a pantheon") {
-            choosePantheonBelief(selectedPantheon!!)
+            chooseBeliefs(null, null, listOf(selectedPantheon!!))
```

Comment:
Seeing these use cases, it might be more readable if you re-order the paramteres of `chooseBeliefs` so the beliefs come first, and the icon name last. If you then give both the `religionName` and `iconName` a default of null, it makes these kinds of functions more readable imo